distribution/docs/src/main/resources/_contents/_core-concepts/search-intro-contents.adoc

```diff
@@ -12,8 +12,7 @@ Additionally, any of the types can be combined to create a compound search.
 ==== Text Search
 
 A text search is used when searching for textual information.
-It searches all textual metadata fields by default, although it is possible to refine searches to a text search on a single attribute.
-It is similar to a Google search over the metadata contained in the Catalog.
+It searches all textual data in the catalog, although it is possible to refine searches to a text search on a single metadata attribute.
```

Comment:
Not 100% sure. Unless I'm misunderstanding the context (certainly possible), I don't see how "contextual" fits here. I can look for other examples though.